prov/gni/include/gnix.h

```diff
@@ -524,6 +597,8 @@ struct gnix_fid_av {
 	size_t capacity;
 	/* How many address are currently stored in AV */
 	size_t count;
+	uint64_t rx_ctx_bits;
+	uint64_t mask;
```

Comment:
Since you are touching this structure, it would be nice to fill out the doxygen comment block for it.